src/django_components/apps.py

```diff
@@ -14,7 +14,7 @@ def ready(self) -> None:
         from django_components.component_registry import registry
         from django_components.components.dynamic import DynamicComponent
         from django_components.template_loader import get_component_dirs
-        from django_components.utils import search_dirs, watch_files_for_autoreload
+        from django_components.util.utils import search_dirs, watch_files_for_autoreload
```

Comment:
Maybe call utils something else when you are there messing about? util.utils doesn't look good :)

Reply:
Agree, but wasn't sure what other name to give it. Maybe can rename it to `misc.py`, so it'd be `util.misc`?

As for the idea of splitting utils into (i assume) single-function files, I'm not personally a favourite, because I find it harder to find common patterns and navigate at such granular level.